Index: index.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/index.html b/index.html
--- a/index.html	(revision 69858ec17b130bb567ddfec2838a4c7097507e2c)
+++ b/index.html	(date 1744308826777)
@@ -1,0 +1,147 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="CSS/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+
+
+<!-- HEADER -->
+<header>
+    <div class="logo">
+        <h2>Portfolio</h2>
+    </div>
+<nav>
+    <a href="index.html">home</a>
+    <a href="contact.html">contact</a>
+</nav>
+</header>
+<!-- HEADER -->
+
+
+
+
+<!--MAIN -->
+<main>
+<!--  INTRO   -->
+    <div class="intro">
+  <!--PHOTO-->
+        <div>
+           <img src="img/myphoto.jpg" ALT="profile picture" width="430" height="480">
+        </div>
+<!--PHOTO END-->
+
+        <div>
+     <!--ARCTICLE-->
+            <div>
+          <!-- WELCOM -->
+               <h3>
+               Full Stack Web Developer JS, React, Node.js, Express, MongoDB
+               </h3>
+           <!--WRITIN-->
+               <p>
+               Hello my name is Murat! i study as a software engineering student at University of Sabahattin Zaim.
+               I study even in website Codecademy. I know a lot of technologies like React, Node.js, Express, MongoDB.
+               I have many projects in my GitHub. You can view my some projects here. <br>
+                   <br>
+                   <label>Number Guesser!</label><br>
+                   <button>
+                       <a href="https://wordpress-developer01.github.io/codecademy1/">view project</a>
+                   </button> <br>
+                  <label>PYRAMIDS</label><br>
+                   <button>
+                       <a href="https://github.com/wordpress-developer01/JS-PYRAMIDS">view project</a>
+                   </button> <br>
+                   <label>Project Etch-a-Sketch</label><br>
+                   <button>
+                       <a href="https://wordpress-developer01.github.io/Project-Etch-a-Sketch/">view project</a>
+                   </button> <br>
+                   <label>getPlay</label><br>
+                   <button>
+                       <a href="https://github.com/wordpress-developer01/getPlay-1">view project</a>
+                   </button> <br>
+                   <label>Sleep-Debt-Calculator</label><br>
+                   <button>
+                       <a href="https://github.com/wordpress-developer01/Sleep-Debt-Calculator">view project</a>
+                   </button> <br>
+                   <label>Dog-Years</label><br>
+                   <button>
+                       <a href="https://github.com/wordpress-developer01/Dog-Years">view project</a>
+                   </button> <br>
+                   <label>project-weather</label><br>
+                   <button>
+                       <a href="https://github.com/wordpress-developer01/project-weather">view project</a>
+                   </button> <br>
+               </p>
+                
+            </div>
+
+       </div>
+        <!--ARTICLE END-->
+    </div>
+<!-- INTRO END   -->
+
+    <div class="projects">
+        <div>
+            Project 1: big-project.js
+            Description:
+            This project is a motivational message generator built for a portfolio assignment on Codecademy. <br>
+            It demonstrates core JavaScript concepts such as array manipulation, functions, and random selection logic.
+
+            <br>What it does:
+
+            Randomly selects one phrase from each of the three arrays: firstly, secondly, and thirdly;
+            Combines them into a complete <br> motivational message;
+            Logs the result to the console;
+            Each execution gives a new, unique message.
+
+            <br><button id="install"><a>view project</a></button>
+        </div>
+
+<!--<div></div>-->
+<!--<div></div>-->
+<!--<div></div>-->
+<!--<div></div>-->
+<!--<div></div>-->
+<!--<div></div>-->
+<!--<div></div>-->
+<!--<div></div>-->
+<!--<div></div>-->
+    </div>
+
+</main>
+<!-- MAIN-->
+
+
+
+
+<!-- FOOTER -->
+<footer>
+
+    <form action="JS/contact.php" method="post" id="contact-form">
+        <label for="fname">First name:</label><br>
+        <input type="text" id="fname" name="fname" value="John"><br>
+        <label for="lname">Last name:</label><br>
+        <input type="text" id="lname" name="lname" value="Doe"><br>
+        <label for="email">Email:</label><br>
+        <input type="email" id="email" name="email" value="john@doe.com"><br>
+        <label for="message">Message:</label><br>
+        <textarea id="message" name="message" rows="4" cols="50"></textarea>
+        <p id="form-status"></p>
+        <button type="submit">Send</button>
+    </form>
+
+</footer>
+<!--FOOTER -->
+
+<script src="JS/index.js"></script>
+</body>
+</html>
\ No newline at end of file
Index: JS/index.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/JS/index.js b/JS/index.js
--- a/JS/index.js	(revision 69858ec17b130bb567ddfec2838a4c7097507e2c)
+++ b/JS/index.js	(date 1744306051940)
@@ -1,0 +1,21 @@
+document.getElementById('install').addEventListener('click', function() {
+    const link = document.createElement('a');
+    link.href = "files/code-projects-main.zip";
+    link.download = "code-projects-main.zip";
+    link.click();
+});
+
+document.getElementById('contact-form').addEventListener('submit', function(e) {
+    e.preventDefault();
+      const name = document.getElementById('fname').value.trim();
+      const lastName = document.getElementById('lname').value.trim();
+      const email = document.getElementById('email').value.trim();
+      const message = document.getElementById('message').value.trim();
+      const status = document.getElementById("form-status");
+
+      if(!name || !lastName || !email || !message) {
+          status.textContent = "Пожалуйста, заполните все поля.";
+          status.style.color = "red";
+          return;
+      }
+});
Index: CSS/index.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/CSS/index.css b/CSS/index.css
--- a/CSS/index.css	(revision 69858ec17b130bb567ddfec2838a4c7097507e2c)
+++ b/CSS/index.css	(date 1744309380406)
@@ -1,0 +1,91 @@
+* {
+    font-family: "Roboto", sans-serif;
+}
+
+body {
+    background-color: white;
+    color: black;
+    transition: background-color 0.3s, color 0.3s;
+    margin: 0;
+    padding: 0;
+}
+
+
+header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.logo{
+    margin-left: 130px;
+}
+
+a {
+    text-decoration: none;
+    text-transform: uppercase;
+    color: black;
+    margin-left: 10px;
+}
+
+
+nav{
+    margin-right: 80px;
+    background-color: orange;
+    padding: 10px;
+    border-radius: 10px;
+}
+
+
+img{
+    margin-left: 100px;
+    margin-top: 70px;
+}
+
+.intro {
+    display: flex;
+    gap: 50px;
+}
+
+.projects {
+    margin-top: 40px;
+    margin-left: 100px;
+}
+
+button{
+    border: none;
+    outline: none;
+    background-color: orange;
+    border-radius: 10px;
+    margin-top: 10px;
+    padding: 10px;
+}
+
+input {
+    border-radius: 10px;
+    width: 200px;
+    height: 30px;
+}
+
+footer {
+    padding: 0px;
+    color: white;
+    background-color: black;
+}
+
+
+
+form {
+    margin-top: 50px;
+    padding-top: 20px;
+    margin-left: 650px;
+    border-radius: 0px;
+    width: 300px;
+    padding-bottom: 50px;
+}
+
+#form-status {
+    margin-top: 10px;
+    font-size: 14px;
+    text-align: center;
+}
\ No newline at end of file
Index: style.css
===================================================================
diff --git a/style.css b/style.css
deleted file mode 100644
--- a/style.css	(revision 69858ec17b130bb567ddfec2838a4c7097507e2c)
+++ /dev/null	(revision 69858ec17b130bb567ddfec2838a4c7097507e2c)
@@ -1,176 +0,0 @@
-:root{
-    --color-1: #aa9e54;
-    --color-2: #D27487;
-    --color-3: #4A2B33;
-    --color-4: #E9E9E3;
-    --color-5: red;
-}
-
-
-
-*{  font-family: 'Open Sans', sans-serif;
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
-}
-
-/*  HEADER  */
-header{
-    position: fixed;
-    width: 100%;
-    height: 90px;
-    background-color: #4A2B33;
-}
-
-.logo{
-    font-size: 24px;
-    color: #D27487;
-    margin-left: 450px;
-}
-
-nav{
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    padding-top: 15px;
-}
-
-ul{
-    display: flex;
-    align-items: center;
-    color: #D27487;
-    gap: 10px;
-}
-
-li{
-    display: flex;
-    gap: 10px;
-    cursor: pointer;
-}
-
-.icons{
-    display: flex;
-    justify-content: end;
-    gap: 10px;
-    margin-left: 500px;
-}
-
-
-.nav-sav{
-   display: none;
-   position: absolute;
-   background-color: var(--color-3);
-   margin-left: 21%;
-   margin-bottom: 25px;
-}
-
-/*  HEADER  */
-
-/*  MAIN  */
-
-.center-photo{
-    width: 100%;
-}
-
-.text{
-    margin: 100px 0px 100px 500px;
-    font-size: 32px;
-}
-
-.main-center{
-    display: flex;
-    background-color:#E9E9E3 ;
-    justify-content: space-between;
-    padding: 70px 80px 50px 80px;
-}
-
-.main-center>div>p{
-    font-size: 32px;
-}
-
-.main-center>div>button{
-    display: flex;
-    font-size: 30px;
-    background-color: #4A2B33;
-    color: #D27487;
-    border: none;
-    border-radius: 10px;
-    padding: 10px 20px;
-    gap: 10px;
-    margin-top: 20px;
-    /* flex-direction: column-reverse; */
-    /* flex-grow: 1; */
-}
-
-
-.main-center-2{
-    background-color: var(--color-1);
-    display: flex;
-    gap: 100px;
-    padding: 100px 100px 100px 100px;
-}
-.main-center-2>div>p{
-    color: #FFFFFF;
-    font-size: 32px;
-    margin-left: 100px;
-}
-
-
-/*  MAIN  */
-
-
-
-.modal {
-    display: none;
-    position: fixed;
-    top: 0;
-    left:0;
-    width: 100%;
-    height: 100%;
-    background-color: rgba(0, 0, 0, 0.5);
-    justify-content: center;
-    align-items: center;
-}
-
-.modal-content {
-    background: white;
-    padding: 20px;
-    border-radius: 5px;
-    width: 300px;
-    text-align: center;
-    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
-    margin: 250px 0px 600px 700px;
-}
-
-
-.btk{
-    all: unset;
-    background-color: #4A2B33;
-    color: #D27487;
-    width: 95%;
-    border-radius: 10px;
-    margin-top: 8px;
-    cursor: pointer;
-}
-
-.X{
-    all: unset;
-    background-color: #4A2B33;
-    color: #D27487;
-    border-radius: 50%;
-    padding: 5px;
-    cursor: pointer;
-    width: 15px;
-    height: 20px;
-    margin-left: 90%;
-}
-
-
-.facebook{
-    width: 150px;
-    height: 120px;
-    position: fixed; /* Фиксированная позиция относительно окна */
-    z-index: 1000; 
-    bottom: 20px; 
-    right: 20px;
-}
